Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN apt-get update && \
         sudo \
         libstdc++6 \
         zlib1g \
+        sox \
```

Comment:
Nice change!

This is probably the reasons dev's never noticed things like #277 before.